packages/studio/styled-system/css/css.d.ts

```diff
@@ -1,9 +1,3 @@
 /* eslint-disable */
 import type { SystemStyleObject } from '../types'
-
-interface CssFunction {
-  (styles: SystemStyleObject): string
-  raw: (styles: SystemStyleObject) => SystemStyleObject
-}
-
-export declare const css: CssFunction;
\ No newline at end of file
+export declare function css(styles: SystemStyleObject): string
```

Comment:
you probably need to `git rebase origin/main`

Reply:
Yeah I did that and still... something weird happened here